### PR TITLE
Portfolio Project - Desktop version

### PR DESCRIPTION
**Pull request by dselasea**
In this pull request I
- Wrote media queries for desktop version
- I made the UI responsive
- I wrote media queries to support different screen sizes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,11 +17,11 @@
...
-                        <a href="# ">Contact</a>
-                    </div>
+                    <ul class="nav__links-items">
+                        <li><a href="#">Portfolio</a></li>
+                        <li><a href="#">About</a></li>
+                        <li><a href="# ">Contact</a></li>
+                    </ul>
                     <div class="nav__hamburger">
                         <img src="./images/Enabled@2x.svg" alt="hamburger menu">
                     </div>
```
Comment (amejid) [inline index.html:27]: - [ ] Good job with the design. Kindly update the `font-size` and `font-weight` of the `h1` `Hey there. I'm...` to match the Figma template. 🤝️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||

File: style.css [hunk 2]
```diff
@@ -274,3 +284,142 @@ textarea {
...
+    display: flex;
+  }
+
+  .nav__links-items a {
+    color: var(--dark);
+    font-weight: bold;
+  }
+
+  header {
+    background-image: url(./images/header-illustration-desktop.svg);
```
Comment (amejid) [inline style.css:312]: - [ ] Currently, the background images are not getting displayed. I don't think the images are uploaded to the repository. Please look into the issue and update it accordingly. 🧐️

**Your Design**

Screenshot from 2023-01-20 09-03-41

File: style.css [hunk 3]
```diff
@@ -274,3 +284,142 @@ textarea {
...
+
+/* Desktop styles */
+@media screen and (min-width: 840px) {
+  .projects {
+    grid-template-columns: repeat(2, 1fr);
+    gap: 2rem;
+  }
+
+  .about {
+    background: url(./images/illustration-about-me-2.svg) 0 50% no-repeat, url(./images/illustration-about-me.svg) 100% 50% no-repeat;
```
Comment (amejid) [inline style.css:357]: - [ ] The same thing here. 🧐️

**Your Design**

Screenshot from 2023-01-20 09-03-27

File: style.css [hunk 4]
```diff
@@ -274,3 +284,142 @@ textarea {
...
+  }
+
+  .skills__content {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 2rem;
+  }
+
+  .contact {
+    background: url(./images/illustration-desktop-1.svg) 0 100% no-repeat, url(./images/illustration-desktop-2.svg) 25% 80% no-repeat, url(./images/illustration-desktop-3.svg) 100% 50% no-repeat;
```
Comment (amejid) [inline style.css:388]: - [ ] Same thing here. 🧐️

**Your Design**

Screenshot from 2023-01-20 09-03-08

File: index.html [hunk 5]
```diff
@@ -123,101 +124,120 @@ <h4 class="card__title">Multi-Post Stories Gain+Glory</h4>
...
+                <div class="contact__form-desktop">
+                    <form action="https://formspree.io/f/mrgdrewl"
+                    method="POST">
+                        <div class="contact__name">
+                            <div class="form-control">
+                                <input name="first_name" type="text" placeholder="First Name" required maxlength="30">
+                        </div>
+                        <div class="form-control">
+                                <input name="last_name" type="text" placeholder="Last Name" required maxlength="30">
+                        </div>
```
Comment (amejid) [inline index.html:213]: - [ ] I know we have been asking you to stick to the Figma design all time around, but here comes an exception. 😎️
According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md) we are supposed to ignore the Figma design ordering us to create `First name` and `Last name` and replace it with a `Full name` so we can use the same form on multiple displays. 🤝️
Kindly remove the two inputs and replace them with a `Full name` input that takes the entire space taken by both of them.

**Project Requirements**

Screenshot from 2023-01-20 09-07-50

**Review by amejid — CHANGES_REQUESTED** [review]
Hi @dselasea, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linter checks passing :heavy_check_mark: 
- Nice UI design :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi Student,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**